tests/puppeteer/pages/BO/international/currencies/add.js

```diff
@@ -0,0 +1,55 @@
+require('module-alias/register');
+const BOBasePage = require('@pages/BO/BObasePage');
+
+module.exports = class AddCurrency extends BOBasePage {
+  constructor(page) {
+    super(page);
+
+    this.pageTitle = 'Currencies • ';
+
+    // Selectors
+    this.currencySelect = 'span[aria-labelledby=\'select2-currency_selected_iso_code-container\']';
+    this.selectResultOption = 'li.select2-results__option:nth-child(%ID)';
+    this.alternativeCurrencyCheckBox = '#currency_unofficial + i';
+    this.currencyNameInput = '#currency_names_%ID';
+    this.symbolInput = '#currency_symbols_%ID';
+    this.isoCodeInput = '#currency_iso_code';
+    this.exchangeRateInput = '#currency_exchange_rate';
+    this.decimalsInput = '#currency_precision';
+    this.statusSwitch = 'label[for=\'currency_active_%ID\']';
+    this.resetDefaultSettingsButton = '#currency_reset_default_settings';
+    this.saveButton = 'div.card-footer button[type=\'submit\']';
+
+    // currency modal
+    this.currencyLoadingModal = '#currency_loading_data_modal';
+  }
+
+  /*
+  Methods
+   */
+
+  /**
+   * Add official currency
+   * @param currencyData, currency to add
+   * @return {Promise<textContent>}, successful text message that appears
+   */
+  async addOfficialCurrency(currencyData) {
+    // Select currency
+    await this.page.select('#currency_selected_iso_code', currencyData.isoCode);
```

Comment:
This selector shouldn't be set elsewhere? :thinking:

Reply:
nice catch, fixed